commands/commandargs.go

```diff
@@ -4,18 +4,53 @@
 package commands
 
 import (
+	"strings"
+
 	"github.com/mattermost/mmctl/client"
 
 	"github.com/mattermost/mattermost-server/v5/model"
 )
 
-// getCommandFromCommandArg retrieves a Command by command id. Future versions
-// may allow lookup by team:trigger
+// getCommandFromCommandArg retrieves a Command by command id or team:trigger.
 func getCommandFromCommandArg(c client.Client, commandArg string) *model.Command {
 	if checkSlash(commandArg) {
 		return nil
 	}
 
-	cmd, _ := c.GetCommandById(commandArg)
+	cmd := getCommandFromTeamTrigger(c, commandArg)
+	if cmd == nil {
+		cmd, _ = c.GetCommandById(commandArg)
+	}
+	return cmd
+}
+
+// getCommandFromTeamTrigger retrieves a Command via team:trigger syntax.
+func getCommandFromTeamTrigger(c client.Client, teamTrigger string) *model.Command {
+	arr := strings.Split(teamTrigger, ":")
+	if len(arr) != 2 {
+		return nil
+	}
+
+	team, _ := c.GetTeamByName(arr[0], "")
+	if team == nil {
+		return nil
+	}
+
+	trigger := arr[1]
+	if len(trigger) == 0 {
+		return nil
+	}
+
+	var cmd *model.Command
+	list, _ := c.ListCommands(team.Id, false)
+	if list == nil {
+		return nil
+	}
+
+	for _, c := range list {
+		if c.Trigger == trigger {
+			cmd = c
+		}
+	}
 	return cmd
```

Comment:
We can return directly in the loop to avoid traversing the whole command list if we've found the one we are looking for:

```suggestion
	list, _ := c.ListCommands(team.Id, false)
	if list == nil {
		return nil
	}

	for _, cmd := range list {
		if cmd.Trigger == trigger {
			return cmd
		}
	}
	return nil
```

(name of the command in the loop changed from `c` to `cmd` to avoid reusing `c` in the function)